src/app/public/modules/dropdown/dropdown.component.ts

```diff
@@ -430,7 +430,7 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
         case SkyDropdownMessageType.Reposition:
           // Only reposition the dropdown if it is already open.
           /* istanbul ignore else */
-          if (this.isOpen) {
+          if (this.isOpen && this.affixer) {
```

Comment:
Fix for #137

Reply:
Can we write a unit test to check for the scenario we fixed?